src/pages/index.js

```diff
@@ -4,114 +4,283 @@ import FormValidator from "../components/FormValidator.js";
+      })
+      .catch((err) => {
+        alert(`${err} Failed to add/remove like.`);
+      });
+  }
 }
 
-function renderCard(data, placement = "prepend") {
-  const cardElement = createCard(data);
-  cardListEl[placement](cardElement);
+function handleDeleteClick(card) {
+  confirmDeletePopup.open();
+  confirmDeletePopup.setSubmitAction(() => {
+    confirmDeletePopup.setDeleteLoading(true);
+    api
+      .deleteCard(card._id)
+      .then((result) => {
+        card.handleDeleteCard(result);
+        confirmDeletePopup.close();
+      })
+      .catch((err) => {
+        alert(`${err} Failed to delete post.`);
+      })
+      .finally(() => {
+        confirmDeletePopup.setDeleteLoading(false);
+      });
+  });
 }
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (data) => {
-      //get the card html element
-      const cardElement = createCard(data);
-      //stick the card element on the page
-      cardSection.addItem(cardElement, "append");
-    },
-  },
-  ".cards__list"
-);
-cardSection.renderItems();
+/* -------------------------------------------------------------------------- */
+/*                                  Functions                                 */
+/* -------------------------------------------------------------------------- */
+
+function handleImageClick(card) {
+  previewImagePopup.open(card);
+}
+
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleDeleteClick,
+    handleAddLike
+  );
+  return card.getView();
+}
 
-/* Event Listeners */
+/* -------------------------------------------------------------------------- */
+/*                               Event Listeners                              */
+/* -------------------------------------------------------------------------- */
 
-profileEditButton.addEventListener("click", () => {
-  const { name, description } = userInfo.getUserInfo();
+avatarImage.addEventListener("click", () => {
+  avatarEditForm.open();
+});
+
+pencil.addEventListener("click", () => {
+  avatarEditForm.open();
+});
+
+addNewCardBtn.addEventListener("click", () => {
+  addCardFormValidator.resetValidation();
+  addImageForm.open();
+});
+
+profileEditBtn.addEventListener("click", () => {
+  const { name, description } = profileUserInfo.getUserInfo();
   profileTitleInput.value = name;
-  profileDescriptionInput.value = description;
-  editCardFormValidator.resetValidation();
-  profileEditPopup.open();
+  profileSubtitleInput.value = description;
+  profileEditForm.open();
+});
+
+modalImage.addEventListener("click", () => {
+  previewImagePopup.open();
+});
+
+/* -------------------------------------------------------------------------- */
+/*                                   Classes                                  */
+/* -------------------------------------------------------------------------- */
+
+const profileEditImage = new FormValidator(config, editImageForm);
```

Comment:
You've created 2 `avatar` form validators: `profileEditImage` and `avatarEditFormValidation`. Please, delete one of them